tests/robustness/snapif/kopia_snapshotter.go

```diff
@@ -0,0 +1,170 @@
+package snapif
+
+import (
+	"crypto/sha1"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	kopiarun "github.com/kopia/kopia/tests/tools/kopia_runner"
+)
+
+var _ Snapshotter = &KopiaSnapshotter{}
+
+// KopiaSnapshotter implements the Snapshotter interface using Kopia commands
+type KopiaSnapshotter struct {
```

Comment:
Is it desirable to move this implementation to the `kopia_runner` package instead? It would still comply with the expected interface used by other parts of the framework.

Reply:
yep sure, that will work. Makes sense to adorn the kopia runner with a bunch of helper methods

Reply:
It can be a separate struct the way it is, just move the implementation in this file to the other package.